### ISSUE TITLE
Bug report: Accessibility of the map 

### ISSUE THREAD

**Issue by Daniel-UCL** [Daniel-GDI-Hub/data-portal#12]
## The problem
The screen reader accessibility on Chrome is terrible. Some unlabelled image elements appear, placing a barrier to navigation.

## Help us reproduce the problem

**Steps to reproduce the behaviour:**

1. Go to Map

**Add screenshots**

**Desktop (please complete the following information):**

 - OS: Mac OS
 - Browser Chrome
 - Version 

**Smartphone (please complete the following information):**

**Additional context**
tested with voice over screenreader on Mac. 

## The expectation

The visual map should be either hidden or clutter free to navigate with a screenreader.

**Comment by Saferrei** [comment]
Images on the website such as the lightbulb in the bottom section of the home page are no longer described with voiceover on Chrome. It skips from the top heading to the next as if the image wasn't there.

**Comment by Daniel-UCL** [comment]
The map is still not accessible (no known good solution), so we decided to apply an aria-hide role to the map area, such that at least it does not get in the way of screen reader users. The data by country and data by AT pages are good alternatives for screen reader users to interact with the data.
